tests/test_features/test_importances.py

```diff
@@ -256,12 +258,11 @@ def test_multi_coefs_stacked(self):
         """
         Test stack plot with multidimensional coefficients
         """
-        X_iris, y_iris = load_iris(True)
-        X_iris_pd = pd.DataFrame(X_iris, columns=['f1', 'f2', 'f3', 'f4'])
+        X, y = load_iris(True)
 
-        viz = FeatureImportances(LogisticRegression(), stack=True)
-        viz.fit(X_iris_pd, y_iris)
-        viz.poof()
+        viz = FeatureImportances(LogisticRegression(random_state=222), stack=True)
+        viz.fit(X, y)
+        viz.finalize()
```

Comment:
Don't use pandas in this test to ensure complete test compatibility.

Reply:
I thought we should test for pandas per Audit

Reply:
You're right of course, in this case we do have a test for Pandas for this visualizer on [L271](https://github.com/DistrictDataLabs/yellowbrick/blob/08ee4e41bdb25bdc24f32904dc31f771676f0362/tests/test_features/test_importances.py#L271).  The pandas test is a regression test to make sure that our tests work with a data frame and a non-random example. We don't have to have a matrix of pandas/non-pandas for every test, and in fact, there should really be only 1 pandas test that doesn't test anything else per visualizer since it injects more complexity into the test. 

In this case I took off pandas because we already had a pandas test and because the test didn't have a `@skipif(pd is None, reason='pandas required for this test)`; Pandas didn't add anything to the test so it was removed!

Reply:
understood...I totally missed L271

Reply:
No worries - I'm glad you asked!